assets/src/edit-story/components/library/panes/text/textSets/textSetsPane.js

```diff
@@ -92,9 +92,23 @@ function TextSetsPane({ paneRef }) {
 
   const addIdToTextSets = useCallback(
     (textSetsArray = []) =>
-      textSetsArray.map((elements) => {
+      textSetsArray.map(({ textSetFonts = [], textSetCategory, elements }) => {
         return {
           id: `text_set_${uuidv4()}`,
+          title: sprintf(
+            /* translators: 1: text set title. 2: text set category. */
+            _n(
+              'Text set %1$s with %2$s font',
+              'Text set %1$s with %2$s fonts',
+              textSetFonts.length,
+              'web-stories'
+            ),
+            textSetCategory,
+            textSetFonts.join(
+              /* translators: delimiter used in a list */
+              __(', ', 'web-stories')
+            )
+          ),
```

Comment:
@swissspidy - can I get your eyes on this i18n set up if you have a second to verify that it's right?

Reply:
The translator comment says "text set category" but the string actually contains a list of fonts (`%2$s font` / `%2$s fonts`). So I'd update the translator comment to `/* translators: 1: text set title. 2: list of fonts. */`

Reply:
gosh dang it i was so focused on the translation i forgot to proof read. thanks 🤦‍♀️